### DISCUSSION TITLE
Initial values of attributes

### DISCUSSION THREAD

**Discussion by MeteorTony** [CastleLab/COMP3021-2021Fall-PA1-Student-Version#44] [Q&A] [answered]
Some attributes are not mentioned in the instructions of building constructors.
For example, player in GameBoard(..), initialNumOfGems in GameState(...)
So what supposed should be their value to be assigned to in constructors?

**Comment by Derppening** [comment, marked as answer]
If you don't find these instance variables to be useful, don't use them.

I will only check whether your implementation of the `public` methods are correct.
